### PR TITLE
Kelsonic 25001 safari button outline

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/25001

This PR removes the transition property from the back to top button.

**Related PRs:**

https://github.com/department-of-veterans-affairs/vets-website/pull/18065
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/649

## Testing done
https://github.com/department-of-veterans-affairs/va.gov-team/issues/25001#issuecomment-889413479

## Screenshots
https://github.com/department-of-veterans-affairs/va.gov-team/issues/25001#issuecomment-889413479

## Acceptance criteria
- [x] Back to top button should show outline when focused on Safari

## Definition of done
- [x] Changes have been tested in vets-website
- [x] Changes have been tested in IE11, if applicable
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Rleahy22 — APPROVED** [review]

**Review by alexandec — APPROVED** [review]

**Comment by alexandec** [comment]
PR seems fine, but the new formation version doesn't seem to exist (yet?)

**Comment by kelsonic** [comment]
@alexandec Correct 🙂 Related PR: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/649

**Review by bkjohnson — DISMISSED** [review]
I'd like to block this for now until we have a better understanding of why the `gi-sandbox` BTT button has deviated from the design system.

**Review by bkjohnson — COMMENTED** [review]
My goal here isn't actually to prevent this PR from being merged - I just wanted to jump in and ask a question _before_ this gets merged and nobody ever looks at it again :upside_down_face: . Since I don't want to block things from happening if someone approves this before I'm awake tomorrow, I'm dismissing the change request.

However, I would still like to understand what prompted the BTT copy + modification to happen rather than contributing any fixes/enhancements to the design system. I remember some discussion around this and [thought the change might be limited to `z-index`](https://dsva.slack.com/archives/C0NGDDXME/p1626721673280100?thread_ts=1626701433.264600&cid=C0NGDDXME), but I did a diff between appropriate files and it looks more complicated than that. #17930 also doesn't really explain the Back to Top changes.


When you approve, please let us know so that the design system team can address the problem that interfered with a system contribution.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@bkjohnson For https://github.com/department-of-veterans-affairs/vets-website/pull/17930 I attempted to use the code provided at https://design.va.gov/components/back-to-top and it did not work out of the box within the CT Sandbox application.  See discussion in https://dsva.slack.com/archives/C0NGDDXME/p1626701433264600 

There were several issues that made it not work as needed:
- There are 2 sticky/floating components within the Sandbox; the compare drawer and "Back to top".  See https://staging.va.gov/gi-bill-comparison-tool-sandbox/profile/31005740 and scroll down past header/jump links
- The compare drawer changes heights when expanded vs collapsed so need to account for "Back to top" placement because of this
- The "Back top" was not floating right correctly within the page, it was either not going right or too far to the right. The hack in https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi-sandbox/components/BackToTop.jsx is too grab the parent element's x coordinate and since the margin on both sides is the same use that to determine the value for the css `right` property of the button

Since the changes made to the CT Sandbox "Back to top" were not pure css changes I did not request the changes be incorporated into the design system.  

The [CT Sandbox](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/gi-sandbox) and [CT Classic](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/gi) have several components and elements that are based on the design system but modified to work with the CT application.

**Review by zurbergram — CHANGES_REQUESTED** [review]
Please go to http://localhost:3001/gi-bill-comparison-tool-sandbox/profile/31005740 on this branch locally and provide screenshots that the "Back to top" button is correctly positioned on bottom right of screen above the compare drawer in both expanded and collapsed state

See https://staging.va.gov/gi-bill-comparison-tool-sandbox/profile/31005740 for current functionality.

**Comment by kelsonic** [comment]
Sorry not sure what all the comments are referring to, this PR has to be merged first to remove the transition: all for buttons on safari due to an a11y error: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/649 `"@department-of-veterans-affairs/formation": "6.16.3"` is not currently published until the related PR is merged + deployed to NPM, apologies for any confusion! 🙂

**Review by zacharymorel — APPROVED** [review]
Weird that transitions prevent focus on element in safari. But we got to do what we got to do for a11y.

**Review by zurbergram — APPROVED** [review]

**Comment by kelsonic** [comment]
@bkjohnson @cvalarida May I have a review?

**Comment by kelsonic** [comment]
@bkjohnson @cvalarida May I have a review? Any success deploying the latest version of formation?

**Review by bkjohnson — COMMENTED** [review]
I just published the latest version of formation.

Also, since this is app code, there's nothing for us to review.

**Comment by kelsonic** [comment]
@bkjohnson Thanks 🙂

**Comment by bkjohnson** [comment]
Thank you for that! We will make an effort to follow up elsewhere now that we have some information, but app code isn't our responsibility.

**Pull request merged**
